reference/checkouts.sf.yml

```diff
@@ -2288,65 +966,90 @@ components:
           description: Coupons applied at the checkout level.
           items:
             $ref: '#/components/schemas/CheckoutCoupon'
-        orderId:
+        createdTime:
           type: string
-          description: ''
-          nullable: true
-        shippingCostTotal:
-          type: number
-          description: Shipping cost before any discounts are applied.
-          format: float
+          description: Time when the cart was created.
+        customer: 
+          $ref: '#/components/schemas/Customer'
+        customerMessage:
+          type: string
+          description: Shopperʼs message provided as details for the order to be created from this cart
+        fees: 
+          type: array
+          items:
+            type: object
+            properties: {}
+        giftCertificates:
+          type: array
+          description: Applied gift certificate (as a payment method).
+          items:
+            $ref: '#/components/schemas/checkoutGiftCertificates'
         giftWrappingCostTotal:
           type: number
           description: Gift wrapping cost for all items, including or excluding tax.
+        grandTotal:
+          type: number
+          description: The total payable amount, before applying any store credit or gift certificate.
+          format: float
         handlingCostTotal:
           type: number
           description: Handling cost for all consignments including or excluding tax.
           format: float
-        taxTotal:
-          type: number
+        id:
+          type: string
           description: ''
-          format: float
-        taxes:
+          format: uuid
+        isStoreCreditApplied:
+          type: boolean
+          description: |
+            `true` value indicates StoreCredit has been applied.
+        orderId:
+          type: string
+          description: ''
+          nullable: true
+        outstandingBalance:
+          type: number
+          description: |
+            `grandTotal` subtract the store-credit amount
+        payments: 
```

Comment:
I need the schema for this field

Reply:
```
           'providerId': string
            'gatewayId': null
            'providerType': 'PAYMENT_TYPE_HOSTED'
            'detail' => [
                'step': string / enum ('FINALIZE', 'INITIALIZE', 'ACKNOWLEDGE')
            ]
```

Reply:
Thanks, I added the schema, if you could check the descriptions I added, that would be great